app/helpers/repos_helper.rb

```diff
@@ -0,0 +1,6 @@
+module ReposHelper
+  def wanna_triage_button(repo, html_class: "")
+    path = user_signed_in? ? repo_subscriptions_path(repo_id: repo.id) : user_omniauth_authorize_path(:github)
+    button_to "I Want to Triage #{repo.path}", path, class: "button #{html_class}"
+  end
+end
```

Comment:
I think add the changes mentioned by @prathamesh-sonpatki and add a newline at the end of the file, and we'll be good.